backend/app/models/enrollment/hubee_portail.rb

```diff
@@ -6,9 +6,6 @@ def submit_validation
     errors.add(:cgu_approved, :invalid, message: "Vous devez valider les modalités d’utilisation avant de continuer") unless cgu_approved?
     errors.add(:dpo_is_informed, :invalid, message: "Vous devez confirmer avoir informé le DPD de votre organisation avant de continuer") unless dpo_is_informed?
     team_members_validation("responsable_metier", "administrateur métier")
-
-    if scopes.empty?
```

Comment:
So you decided to completely erase this validation ? Are you sure we don't need it anymore ?

Reply:
Well, we only have one scope (certDC) and this scope is disabled if one enrollment has been validated. So it seems to be redundant. 
The new error message will be more explicit, I think.

We will need it again, if we will have several scopes for `hubee_portail` and need it for hubee_portail_dila`